src/NUnitFramework/framework/Assert.cs

```diff
@@ -51,11 +53,14 @@ public partial class Assert
         #region Constructor
 
         /// <summary>
+        /// Initializes a new instance of the <see cref="Assert"/> class. 
```

Comment:
Actually, if you read the rest, this comment line is only technically correct. It's actually not possible to create an instance and we shouldn't confuse the issue by saying so.